languages/csharp/concept-exercises/pattern-matching/PatternMatching.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+//See the Shapes file for the types you'll need to support.
```

Comment:
I'd prefer having the types in the same file. Makes for an easier review of the solution, as well as the fact that the in-browser coding doesn't support multiple files at the moment.